site/app/templates/polls/ViewPoll.twig

```diff
@@ -64,7 +64,7 @@
                                 "style": "display: inline"
                             } only %}
                             {% if option.isCorrect() and poll.isReleaseAnswer()%}
-                                <p id='correct-tag' style="color: #0eed0e; margin-left: 0rem ">correct!</p>
+                                <span id='correct-tag'>&#10004;</span>
```

Comment:
This should be replaced with an icon from [Font Awesome](https://fontawesome.com/icons/check?s=solid&f=classic) since this checkmark character isn't guaranteed to look the same on all platforms.  You can find numerous examples of FA icons across the codebase to reference for this.